.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
I would suggest looking into a [global gitignore](https://help.github.com/en/github/using-git/ignoring-files) file. It's nice not to place `.DS_Store` into all of your committed `.gitignore` files.

Reply:
Yep, this looks better